Automattic/wp-calypso
# Survey: Add enter and exit animations

Related to 

## Proposed Changes

* Render surveys only the content is ready to be rendered
* Add animation when the component is rendered or skipped.


## Testing Instructions
* Make sure you don't have the cookie `migration-survey` in your browser; if you have, please remove it!
* Access any of the migration flows


## Technical Notes
* I am using the "new"  transition view API to improve the exit animation.  This kind of animation usually requires special libs or extraneous code to be handled.  The only lib we have on calypso ( "react-transition-group` ) that could help is pretty old and throwing some errors on development env. 


## Pre-merge Checklist
